shared/src/business/useCases/editDocketEntry/completeDocketEntryQCInteractor.js

```diff
@@ -131,6 +135,10 @@ exports.completeDocketEntryQCInteractor = async (
       editState: '{}',
       relationship: DOCUMENT_RELATIONSHIPS.PRIMARY,
       userId: user.userId,
+      workItem: {
```

Comment:
Moving the comment from the old PR to here! Unit test?

Reply:
added to unit test for workitem leadDocketNumber...thank you Mrin